### DISCUSSION TITLE
이메일 => 아이디로 변경

### DISCUSSION THREAD

**Discussion by Hain-tain** [woowacourse-teams/2024-code-zap#433] [기획 관련]
- 이메일을 그냥 아이디로 변경하기로 하였습니다. 왜냐하면 현재 이메일 인증을 통해 유효한 이메일을 받고 있는 것도 아니고 굳이 이메일 형식으로 아이디를 둘 필요가 없을 것 같다고 생각했기 때문입니다.
- 또한 아이디와 username 이 둘다 유니크하기 때문에 두개를 다 남길 필요가 있나, 하나만 남기자! 라는 의견도 있는데 이에 대해서 어떻게 생각하는지 각자 의견 달아주세요~!!

**Comment by HoeSeong123** [comment]
상관없습니다.

**Comment by Jaymyong66** [comment]
유미도 상관없습니다.

**Reply by Jaymyong66** [reply]
유저네임은 용도가 달라서 따로 둬도 되지 않을까 싶습니다.

**Reply by Jaymyong66** [reply]
만약 아이디와 유저네임이 통합되면 아이디는 한글이 안되는건가요?

**Comment by vi-wolhwa** [comment]
수민상관없습니다.

**Comment by zangsu** [comment]
1. 헤인 말 대로 이메일에 대한 기능이 존재하는 것이 아니기에 존속 여부는 자율적으로 잘 결정해 주신 것 같습니다! 👍 
2. 개인적으론 아이디는 인증에 필요한 정보 / username은 불리는 이름으로 느껴져서 둘 다 사용해도 좋을 것 같은데... 안남겨도 상관 없슴다

**Reply by zangsu** [reply]
헤인 설명 납득했슴다

**Comment by zeus6768** [comment]
아이디 사용 동의합니다.

**Comment by kyum-q** [comment]
닉네임이 유니크하지 않아도 될 것 같다.

- 닉네임 보여주는 값이기 때문에 나중에 다른 사람 템플릿 볼 때 헷갈릴 것 같다는 의견이 있었는데, 그거는 크게 신경 쓰이지 않다고 생각합니다~

 근데 다들 상관없다면 상관 없음~
